components/concept/tabstray/src/main/java/mozilla/components/concept/tabstray/TabsTray.kt

```diff
@@ -63,6 +63,11 @@ interface TabsTray : Observable<TabsTray.Observer> {
      */
     fun onTabsChanged(position: Int, count: Int)
 
+    /**
+     * Called when binding a new item to get if it should be shown as selected or not.
+     */
+    fun isTabSelected(tabs: Tabs, position: Int): Boolean
```

Comment:
```suggestion
    fun isTabSelected(tabs: Tabs, position: Int): Boolean = tabs.selectedIndex == position
```

What do you think about having a default implementation that do the basic check for you? I'm not fully convinced I like default impls like this on an API, but I thought I'd throw the idea out there either way. 😄

Reply:
Thanks!
I also thought about this but saw that other methods from this interface also have simple implementations but not as default.
So mainly just to be consistent with them I put the implementation in `TabsAdapter`.
This simple check is the only reason I went with this Fenix - AC - Fenix loop. The list of tabs is in AC, the behavior for if to show a tab selected or not is in Fenix.

I like the default implementation as that would make this a non-breaking change, what I initially intended, but as you've said maybe there shouldn't be any logic at this level.
I would prefer not to be the first that starts polluting this interfaces 😂, will add the breaking change mention if it's ok with you.